migrations/2023-10-24-030352_change_primary_keys_and_remove_some_id_columns/up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE post_saved
+    DROP COLUMN id,
+    ADD PRIMARY KEY (post_id, person_id);
```

Comment:
Is it possible that old databases could have duplicate entries?

Reply:
The table has `UNIQUE` and `NOT NULL` constraints. I changed the migration to remove them after adding the primary key.

Reply:
`NOT NULL` is no longer removed because it does not show any effect on the schema when using pg_dump.